app/experimenter/experiments/tests/mixins.py

```diff
@@ -58,37 +89,27 @@ def setupMockBugzillaCreationFailure(self):
 
     def setUpMockBugzillaInvalidUser(self):
 
-        def mock_reject_assignee(url, bug_data):
-            if "assigned_to" in bug_data:
-                return self.buildMockFailureResponse()
-            else:
-                return self.buildMockSuccessResponse()
+        self.mock_bugzilla_requests_get.side_effect = [
+            self.buildMockFailureResponse(),
+            self.buildMockSuccessBugResponse(),
+            self.buildMockSuccessBugResponse(),
+        ]
 
-        self.mock_bugzilla_requests_post.side_effect = mock_reject_assignee
+    def setUpMockBugzillaFirstBadTicket(self):
```

Comment:
The body of these helper methods is so short and if they're only used once I'd just move this to the actual test.